### ISSUE TITLE
'whats left to scan' reporting a blank list

### ISSUE THREAD

**Issue by F3Speech** [alterNERDtive/VoiceAttack-profiles#64]
Happening when one a single star in the system,
Have since noticed it once when 2 stars were in close.

**Comment by alterNERDtive** [comment]
> Have since noticed it once when 2 stars were in close.

Ah, good. I was worried I hadn’t understood what happened there. For the record, close or not doesn’t matter for stars. The trigger here is a race condition when you finish the discovery scan (that triggers this) too fast.

That also means that your suggested workaround of “just don’t do it when there’s only one body” unfortunately won’t work. It wouldn’t really have anyway though if I had moved the R2R announcement from the `Discovery scan` to the `Jumped` event since body count is unknown at that point (see #61).

Bottom line is still: this looks like an EDDI issue to me. I suspect that it’s not done getting system data from EDSM yet when this happens, and the `bodies to map` speech responder script doesn’t handle that gracefully.

**Comment by alterNERDtive** [comment]
So current EDDI beta lists this change:

> `Bodies to map` updated to default to the current star system if context variable `eddi_context_system_name` is not set and updated the P() function utilization.

I suspect this bug here might be the result of a race condition that existed before, when the new system name had not been set. So the beta _might_ fix this.

**Comment by alterNERDtive** [comment]
@SiIason have you run into this since? I haven’t, but I haven’t really been looking out for it.

**Comment by alterNERDtive** [comment]
Just ran into this, so it’s definitely still a thing :-/

**Comment by alterNERDtive** [comment]
Anyone still seeing this? I’m not really playing anymore so couldn’t tell.
